lib/model/query/datasets.js

```diff
@@ -290,6 +291,7 @@ const publishIfExists = (formDefId, publishedAt) => async ({ all, context, maybe
       AND dp.name != existing_properties.name
```

Comment:
I'd guess that we don't need this condition anymore. If `dp.name = existing_properties.name`, that would mean that `dp` is published, but we know that's not the case.